quickfix/AttributeRenameDefault.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license Copyright (c) 2014-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+( function() {
+	'use strict'
+
+	CKEDITOR.plugins.a11ychecker.quickFixes.get( {
+		name: 'AttributeRename',
+		callback: function( QuickFix ) {
+			function AttributeRenameDefault(issue) {
```

Comment:
Codestyle issue, no spaces between parenthesis. Do you have linting task hooked to git?